SECURITY.md

```diff
@@ -0,0 +1,5 @@
+# Security Reporting
+
+Thank you in advance for helping us keep RudderStack secure!
+
+Please report any security issues or vulnerabilities to [security@rudderstack.com](mailto:security@rudderstack.com). We will triage the issue and take the necessary steps accordingly.
```

Comment:
```suggestion
Please report any security issues or vulnerabilities to [security@rudderstack.com](mailto:security@rudderstack.com), before proceeding to post them publicly as issues on github or any other public channel (i.e. slack). Other users might be affected by this as well and security vulnerabilities need to be handled quickly and sometimes privately.
We will triage the issue, contact you for further instructions and make sure to take any necessary measures as needed.
```

Let's add some more information to prevent publicising security issues as much as possible. What do you think @ameypv-rudder ?

Reply:
Makes sense @chrikar - slightly modified your suggestion - LMK if it looks okay.

Reply:
I think I prefer the original phrasing in this: `security vulnerabilities need to be handled quickly and sometimes privately`. It's always quickly, sometimes privately :)

Also, let's refrain from using `our` etc in docs in ` our open source community Slack`. Let's use `Rudderstack community Slack` instead.

Thx! 💪🏼

Reply:
@chrikar thanks Christos for your valuable feedback. I've incorporated these changes.